src/style.css

```
@@ -0,0 +1,70 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0;
+}
+
+.bar {
+  margin: 20px 8px;
+}
+
+.search {
+  width: 100%;
+  height: 70px;
+  padding: 5px 62px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 50%;
```

no actual need for the 50%
- only 25px works just the same
```suggestion
  background-position: 25px;
```